config-overrides.js

```diff
@@ -14,6 +15,13 @@ module.exports = {
   webpack: override(
     addBabelPlugin('@emotion/babel-plugin'),
     addWebpackPlugin(new StylelintPlugin({ files: './src/**/*.{tsx,ts}' })),
+    addWebpackPlugin(
+      new BundleAnalyzerPlugin({
+        // use value from environment var STATS or 'disabled'
+        analyzerMode: process.env.STATS || 'disabled',
```

Comment:
What would possible values for `STATS` be?

Reply:
These
> Default: server. In server mode analyzer will start HTTP server to show bundle report. In static mode single HTML file with bundle report will be generated. In json mode single JSON file with bundle report will be generated. In disabled mode you can use this plugin to just generate Webpack Stats JSON file by setting generateStatsFile to true.